docs/user-guide/discovery-and-relay.md

```diff
@@ -9,7 +9,7 @@ Relay node needs to be reachable on a predictable Wireguard port such as 51820 a
 Clone the Nexodus repository on a VM (or bare metal machine). Nexodus relay node must be reachable from all the endpoint nodes that want to join the Nexodus network. Follow the instruction in [Starting The Agent](agent.md#starting-the-agent) section to set up the node and install the `nexd` binary.
 
 ```sh
-sudo nexd --stun relay https://try.nexodus.127.0.0.1.nip.io
+sudo nexd --stun relay --service-url https://try.nexodus.127.0.0.1.nip.io
```

Comment:
I think the CLI lib we use is pretty picky about the order. Since `service-url` is a top level arg, it needs to come before `relay`.

Reply:
yeah your right.  BTW been thinking we should likely avoid configuring the --service-url in most of our examples.  Like the user reading this might be testing against prod, and then it would not be needed.

Reply:
I agree that it's extra noise for most examples.  I'd be all for removing it in most cases.